lib/actions/authActions.js

```diff
@@ -60,3 +53,16 @@ export function signup(newUserData) {
     })
   };
 }
+
+export function validateToken({accessToken, uid, _client}) {
```

Comment:
Oh now I see the `_client` issue. I don't think this will work, because the key for the object we are passing in is still `client` right @dviramontes? We'll need to keep that as client, and name the promise something else. Maybe we could just go with `api`?